### PR TITLE
Updated the PR template

### PR DESCRIPTION
**Pull request by rickard-von-essen**
Reference _GitHub Actions_ instead of _Travis CI_. Also update instructions to use `yarn lint` instead of `make pre-push`.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — COMMENTED** [review]
If we are going to modify this, I think we can remove the latest NOTE. I don't know when this take hours to run on any desktop system :P

**Review by blckmn — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
image

**Pull request merged**
